test/actor.js

```diff
@@ -538,6 +538,31 @@ describe('Actor', function () {
       isStopped(parent).should.be.true;
     });
 
+    it('should be able to access other messages in the queue', async function () {
+      const onCrash = (msg, err, ctx) => {
+        ctx.mailbox.length.should.equal(2);
+        ctx.mailbox[0].message.should.equal('msg1');
+        ctx.mailbox[1].message.should.equal('msg2');
+        return ctx.escalate;
+      };
+      const parent = createSupervisor(system, 'test1');
+      const child = spawn(parent, (state = 0, msg, ctx) => {
+        // Wait for messages to queue up, then fail on msg0
+        if (state + 1 === 1) {
+          delay(100);
```

Comment:
This function needs to be async with delay being awaited for this to function as expected consistently.

Reply:
Fixed.